src/PolicyInsights/PolicyInsights/Cmdlets/Remediation/StartAzureRmPolicyRemediation.cs

```diff
@@ -69,6 +69,18 @@ public class StartAzureRmPolicyRemediation : RemediationCmdletBase
         [ValidateSet("ExistingNonCompliant", "ReEvaluateCompliance")]
         public string ResourceDiscoveryMode { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.RemediationResourceCount)]
+        [ValidateRange(1, int.MaxValue)]
+        public int? ResourceCount { get; set; }
+
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.RemediationParallelDeployments)]
+        [ValidateRange(1, int.MaxValue)]
+        public int? ParallelDeployments { get; set; }
```

Comment:
Is it a better name `ParallelDeployCount`?

Reply:
There are 1116 existing exceptions for this rule across the Azure Powershell repo, many of them are very similar to this case. Why can't we have an exception here as well?

Reply:
Please email to our team channel if you have any question.

Reply:
@wyunchi-ms what is your team channel? couldn't find it in any public\internal docs.

Reply:
@wyunchi-ms nvm. Changed it to `ParallelDeploymentCount`.